core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -233,11 +233,14 @@ protected[core] object WhiskWebActionsApi extends Directives {
 
       val code = fields.get(rp.statusCode).map {
         case JsNumber(c) =>
-          // the following throws an exception if the code is
-          // not a whole number or a valid code
+          // the following throws an exception if the code is not a whole number or a valid code
           StatusCode.int2StatusCode(c.toIntExact)
+        case JsString(s) =>
+          // parse the string to an Int (not a BigInt) matching JsNumber case match above
+          // s.toInt could throw an exception if the string isn't an integer
+          StatusCode.int2StatusCode(s.toInt)
```

Comment:
@tz70s since int2StatusCode itself could throw an exception, and neither exception should be observable, I elided the outer try/catch. I hope you don't mind that I pushed a commit to your branch, figured I'd spare you the small nit.

Reply:
@rabbah, ah, I didn't notice that, thank you!